stac_mjx/controller.py

```diff
@@ -233,17 +233,15 @@ def fit(self, kp_data):
         mjx_data = mjx.com_pos(mjx_model, mjx_data)
 
         # Begin optimization steps
-        # Skip root optimization if model is fixed (no free joint at root)
-        if self._mj_model.jnt_type[0] == mujoco.mjtJoint.mjJNT_FREE:
-            mjx_data = compute_stac.root_optimization(
-                mjx_model,
-                mjx_data,
-                kp_data,
-                self._lb,
-                self._ub,
-                self._body_site_idxs,
-                self._trunk_kps,
-            )
+        mjx_data = compute_stac.root_optimization(
```

Comment:
Why are we removing this?

Reply:
Mistake. Good catch.

Reply:
It's not re-included?

Reply:
Sorry, another mistake. Fixed now. One thing though, should we be doing the same check in fit()?